susemanager/etc/logrotate.d/susemanager-tools

```diff
@@ -11,4 +11,14 @@
     su wwwrun www
 }
 
-
+/var/log/rhn/mgr-create-bootstrap-repo/mgr-create-bootstrap-repo.log {
+    compress
+    dateext
+    daily
+    maxage 30
+    missingok
+    notifempty
+    rotate 30
+    size 4096k
```

Comment:
I think size could make problems.
We want to rotate daily? So no need to specify a size limit.

Reply:
Agreed, I will remove it.